### PR TITLE
License update to dual MIT and Apache 2

### LINKED ISSUE #2552: Add Apache 2.0 license
With [go-ipfs having switched to dual MIT and Apache 2 licenses](https://github.com/ipfs/go-ipfs/issues/6302), can js-ipfs do the same thing as well?

### PR DESCRIPTION
**Pull request by ay2306**
Updated the license as mentioned in issue #2552. This closes #2552

### CODE REVIEW

File: COPYRIGHT [hunk 1]
```diff
@@ -0,0 +1,5 @@
+This project is transitioning from an MIT-only license to a dual MIT/Apache-2.0 license.
+Unless otherwise noted, all code contributed prior to 2019-11-21 and not contributed by
+a user listed in [this signoff issue](https://github.com/ipfs/js-ipfs/issues/2552) is
```
Comment (alanshaw) [inline COPYRIGHT]: I will create a separate issue for this - would you be willing to generate and send me a list of contributors to js-ipfs to date so I can mention them in the issue (like in https://github.com/ipfs/go-ipfs/issues/6302)?

Comment (ay2306) [reply COPYRIGHT]: I will do that.

Comment (ay2306) [reply COPYRIGHT]: @alanshaw Can you please help me in finding the active IRC or gitter where I can ask questions for js-ipfs. I found few but do not know if they are the right ones.

Comment (alanshaw) [reply COPYRIGHT]: I'm on the freenode IRC \#ipfs

**Comment by alanshaw** [comment]
@ay2306 any chance you have time to make the same change to https://github.com/ipfs/js-ipfs-http-client ?

**Comment by ay2306** [comment]
@alanshaw I will make changes to js-ipfs-http-client as well.

**Comment by SidHarder** [comment]
I license past and future contributions under the dual MIT/Apache-2.0 license, allowing licensees to choose either at their option.

**Pull request merged**